OpenRobertaRobot/src/main/java/de/fhg/iais/roberta/syntax/BlockTypeContainer.java

```diff
@@ -117,4 +118,10 @@ public static BlockType getByBlocklyName(String blocklyName) {
         Assert.notNull(blockType, "blockly name is not found: " + blocklyName);
         return blockType;
     }
+
+    @VisibleForTesting
+    public static Map<String, BlockType> getBlockTypesByName() {
```

Comment:
public? why not package visible?

Reply:
ok, was removed later. In general, the cost of checking valis annotations should occur only at server startup time and not for every parse of XML. Or?